src/api/routes/rosetta/construction.ts

```diff
@@ -399,7 +400,7 @@ export function createRosettaConstructionRouter(db: PgStore, chainId: ChainID):
         res.status(400).json(RosettaErrorsTypes.missingTransactionSize);
         return;
       }
-      const feeValue = (BigInt(feeInfo) * BigInt(options.size)).toString();
+      const feeValue = Math.round(Number(feeInfo) * Number(options.size) * 1.5).toString();
```

Comment:
I might be missing something with pox-3, but why is this fee value function being adjusted?

Reply:
OK gotcha, thanks